Blueprints/BlueprintDefinitions/netcore3.1/SimpleS3FunctionServerless/template/src/BlueprintBaseName.1/serverless.template

```diff
@@ -48,7 +48,8 @@
         "Role": null,
         "Policies": [
           "AWSLambda_FullAccess",
-          "AmazonRekognitionReadOnlyAccess"
+          "AmazonRekognitionReadOnlyAccess",
+          "AmazonS3ReadOnlyAccess"
```

Comment:
Can you make these same changes to the F# version of this file.
https://github.com/aws/aws-lambda-dotnet/blob/master/Blueprints/BlueprintDefinitions/netcore3.1/SimpleS3FunctionServerless-FSharp/template/src/BlueprintBaseName.1/serverless.template